### ISSUE TITLE
Homebrew installation error on macOS 10.12

### ISSUE THREAD

**Issue by willjay** [pegasus-isi/homebrew-tools#2]
Hello,

I am a first-time user who just discovered Pegasus.

I am trying to install Pegasus on macOS 10.12 ("Sierra") following the Homebrew formulae posted in the documentation. Unfortunately, I encounter errrors that are somewhat mysterious to me. It looks like perhaps a problem occurs when building the documentation? Anyway, below is my output:

```
$ brew install pegasus htcondor
==> Installing pegasus from pegasus-isi/tools
==> Downloading http://download.pegasus.isi.edu/pegasus/4.6.1/pegasus-source-4.6.1.tar.gz
Already downloaded: /Users/wijay/Library/Caches/Homebrew/pegasus-4.6.1.tar.gz
==> ant clean dist-common doc-manpages
Last 15 lines from /Users/wijay/Library/Logs/Homebrew/pegasus/01.ant:
     [exec] ^
     [exec] compilation error: file http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl line 38 element include
     [exec] xsl:include : unable to load http://docbook.sourceforge.net/release/xsl/current/manpages/lists.xsl
     [exec] http://docbook.sourceforge.net/release/xsl/current/manpages/pi.xsl:1: parser error : Document is empty
     [exec] 
     [exec] ^
     [exec] compilation error: file http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl line 41 element include
     [exec] xsl:include : unable to load http://docbook.sourceforge.net/release/xsl/current/manpages/pi.xsl
     [exec] make: *** [pegasus-config.1] Error 5
     [exec] make: *** [pegasus-configure-glite.1] Error 5

BUILD FAILED
/private/tmp/pegasus-20161006-10885-1tgwaf7/pegasus-source-4.6.1/build.xml:220: exec returned: 2
```

After following the linked troubleshooting [tips](https://git.io/brew-troubleshooting), I'm unfortunately unable to make an progress.

Perhaps you have encountered an issue like this before? Any suggestions?

Thanks,
Will

**Comment by vahi** [comment]
Hi Will 

We will take a look at this and get back to you 

https://jira.isi.edu/browse/PM-1130

Karan

**Comment by mayani** [comment]
@willjay 
We updated a machine to use macOS Sierra. During the process we updated brew to version 1.0.6 by executing  

```
ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
```

Other issue we faced was having to execute 

```
brew unlink asciidoc
brew link --overwrite asciidoc
```

Can you let us know if any of the above resolves your issue? If not, please include outputs of following

```
brew list
brew config
brew doctor
```

**Comment by willjay** [comment]
@mayani 

Thanks for response and the suggestions. Unfortunately, I still get the same error referencing *xsl manpages that I quoted above in my original message.

I was already running Homebrew 1.0.6:

```
$ brew --version
Homebrew 1.0.6
Homebrew/homebrew-core (git revision 8268; last commit 2016-10-07)
```

Here is the output your requested:

```
$ brew list
ant     asciidoc    docbook     gsl     keychain    openssl
```

```
$ brew config
HOMEBREW_VERSION: 1.0.6
ORIGIN: https://github.com/Homebrew/brew
HEAD: 35ee2831086e923e7fcaf75fb440b01312e3f9c5
Last commit: 5 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 8268388604d331d314e586d1b1bda27f626a988d
Core tap last commit: 2 hours ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_REPOSITORY: /usr/local/Homebrew
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com
CPU: quad-core 64-bit haswell
Homebrew Ruby: 2.0.0-p648
Clang: 8.0 build 800
Git: 2.8.4 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Perl: /usr/bin/perl
Python: /usr/bin/python
Ruby: /usr/bin/ruby => /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby
Java: 1.8.0_102
macOS: 10.12-x86_64
Xcode: 8.0
CLT: N/A
X11: N/A
```

```
$ brew doctor
Your system is ready to brew.
```

**Comment by mayani** [comment]
@willjay Can you also attach the entire log file from /Users/wijay/Library/Logs/Homebrew/pegasus/01.ant

**Comment by willjay** [comment]
@mayani 

Attached is the requested file. There seem to be a variety of errors recorded in the file, but I'm not sure the clearest way to proceed.

(I added the *txt extension to get GitHub to accept the attachment, but that's the only change I've made.)

[01.ant](https://github.com/pegasus-isi/homebrew-tools/files/517895/01.ant.txt)

**Comment by mayani** [comment]
@willjay 

The logs don't indicate any error within Pegasus code/docs. So not quite sure why the error is happening.

Some possible solutions - 
1. Noticed you do not have XCode CLT installed.

```
xcode-select --install
brew install htcondor
brew install pegasus
```
1. Noticed a similarly reported error at [https://trac.macports.org/ticket/35303](https://trac.macports.org/ticket/35303)

```
brew install docbook-xsl
brew install htcondor
brew install pegasus
```
1.  Install Pegasus without offending ant target

```
brew install htcondor
brew install pegasus --without-manpages
```
1. Install from installer file at 

[https://download.pegasus.isi.edu/pegasus/4.6.2/pegasus-binary-4.6.2-x86_64_macos_10.dmg](https://download.pegasus.isi.edu/pegasus/4.6.2/pegasus-binary-4.6.2-x86_64_macos_10.dmg)

Let me know if any of the above works for you.

**Comment by willjay** [comment]
@mayani, @vahi 
Using the third suggestion, I was able to install HTCondor and Pegasus successfully:

```
brew install htcondor
brew install pegasus --without-manpages
```

Although it's a bit puzzling that the manpages do not seem to want to install, I suspect it must be some quirk associated with my local configuration. (I did try `brew install docbook-xsl` first, and the installation of `docbook-xsl` exited with success. The subsequent attempt at `pegasus` failed as above.)

Thank you for your time and help.

 I'm content to call the issue resolved at this point.
